packages/monaco/src/browser/style/index.css

```diff
@@ -71,6 +71,14 @@
   height: var(--theia-scrollbar-width) !important;
 }
 
+.monaco-editor .codicon.codicon-debug-start {
+  color: #89d185 !important;
+}
```

Comment:
@msujew I'm not sure about hardcoding values for the icon, can you clarify why it is necessary and where the color comes from?

Reply:
Yeah, totally not necessary. I've added a new variable (`theia-start-debug-color`) to the `debug` css files. I took the color from [here](https://github.com/microsoft/vscode/blob/81626fc25e2ca00cce51f61f6880cfb7c9ff27c5/src/vs/workbench/contrib/debug/browser/debugColors.ts#L24-L28).

I thought about adding `bright` and `dark` css files to monaco, but as the debug package is necessary to display code-lenses, I added the variable in there.

Reply:
I see, thank you! And I kept wondering, where those magic css variables where coming from...

I changed my code to align with the coding guidelines.